app/models/work.rb

```diff
@@ -70,6 +70,7 @@ def create_stat_counter
   # Can't write to work.pseuds until the work has an id
   attr_accessor :new_parent, :url_for_parent
   attr_accessor :new_recipients
+  attr_accessor :new_series
```

Comment:
Is this used anywhere? It's set in the `series_attributes=` function, but I don't see any references to it elsewhere.